images/linux/scripts/tests/Java.Tests.ps1

```diff
@@ -0,0 +1,47 @@
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1" -DisableNameChecking
+
+Describe "Java" {
+
+	if (Test-IsUbuntu20) {
+        $DefaultJavaVersion = "11"
+    }
+
+    if ((Test-IsUbuntu16) -or (Test-IsUbuntu18)) {
+        $DefaultJavaVersion = "8"
+    }
+
+    It "Java <DefaultJavaVersion> is default" -TestCases @{ DefaultJavaVersion = $DefaultJavaVersion } {
+        $actualJavaPath = bash -c "source ${env:INSTALLER_SCRIPT_FOLDER}/../helpers/etc-environment.sh && getEtcEnvironmentVariable JAVA_HOME"
```

Comment:
Why can't we just use `$env:JAVA_HOME` or `Get-EnvironmentVariable "JAVA_HOME"` here?

Reply:
The same below

Reply:
the java-tools installation script set JAVA_HOME variable in /etc/environment and if we want to use this variable we should re-login, but tests are performed right after the installation, so the variable is in /etc/environment file but not in runtime, same as Get-EnvironmentVariable get it from the runtime, but its not loaded yet

Reply:
added update-environment function to eliminate bash -c calls that looks unclear